lib/targets/pods.js

```diff
@@ -44,8 +44,12 @@ module.exports = async (context) => {
     context.logger.info(`Pushing podspec ${fileName} to cocoapods`);
     if (shouldPerform()) {
       const env = _.pick(process.env, ['COCOAPODS_TRUNK_TOKEN', 'PATH']);
-      await spawn(COCOAPODS_BIN, ['repo', 'update'], { cwd: directory, env });
-      await spawn(COCOAPODS_BIN, ['trunk', 'push', fileName], { cwd: directory, env });
+      try {
+        await spawn(COCOAPODS_BIN, ['repo', 'update'], { cwd: directory, env });
+        await spawn(COCOAPODS_BIN, ['trunk', 'push', fileName], { cwd: directory, env });
+      } catch (e) {
+        context.logger.error(`pod command failed with: ${e}`);
```

Comment:
Rethrow the exception here, otherwise the "success" message would be printed, too.